### DISCUSSION TITLE
Artifact Failed when trying to create a helmrelease

### DISCUSSION THREAD

**Discussion by bogdancosmin** [fluxcd/flux2#4445] [Q&A] [answered]
I'm just getting started using FluxCD and I'm trying to deploy RedPanda-console via the Flux CLI
```
flux create source helm redpanda --url https://charts.redpanda.com/ -n kafka

flux create helmrelease redpanda --chart redpanda/console --source HelmRepository/redpanda \
--namespace kafka
```

I double/triple checked that the pathing is correct but I keep getting this error 
```
◎ waiting for HelmRelease reconciliation
✗ client rate limiter Wait returned an error: context deadline exceeded
```
In the helmrelease I get the following error and reason 
```
│     Message:               HelmChart 'kafka/kafka-redpanda-console' is not ready          │
│     Reason:                ArtifactFailed
```

Please help :) !!!

**Comment by hiddeco** [comment]
Can you please share the output of `kubectl describe -n kafka helmchart/kafka-redpanda-console`?

**Reply by bogdancosmin** [reply]
`HelmChart 'kafka/kafka-redpanda' is not ready`

`Error from server (NotFound): helmcharts.source.toolkit.fluxcd.io "kafka-redpanda-console" not found`

**Reply by hiddeco** [reply]
Try again with `kubectl describe -n kafka helmchart/kafka-redpanda`?

**Reply by bogdancosmin** [reply]
`  Warning  InvalidChartReference  26s   source-controller  invalid chart reference: failed to get chart version for remote reference: no chart name found`

**Reply by hiddeco** [reply, marked as answer]
Ah, I see the issue now. You are using the `--chart` variable as if you are using `helm`.

The correct command is: `flux create helmrelease redpanda --chart console --source HelmRepository/redpanda --namespace kafka`.

Unlike `helm`, you are already defining the repository index the chart can be found at using `--source`. Which means you should use _just_ the chart name, and not the name of the repository like you would normally do using `helm` itself.
